tests/e2e/e2e_test.go

```diff
@@ -91,6 +91,22 @@ func SourceTest(appTestName string, sourceType string, source string) {
 	Expect(getDc).To(ContainSubstring(source))
 }
 
+// EnvVarTest checks the component container env vars in the build config for git and deployment config for git/binary/local
+// appTestName is the app of the app
+// sourceType is the type of the source of the component i.e git/binary/local
+func EnvVarTest(appTestName string, sourceType string) {
```

Comment:
Can we make this function generic to accept dc name, key/value as an arguments?

Reply:
But we are checking build config too in the function